### PR TITLE
Add support for Visual Studio 15 Preview 4

### PR DESCRIPTION
**Pull request by tcare**
We'll use the Visual Studio 2015 platform toolset for now, this will just allow us to use the IDE without it complaining.

### CODE REVIEW

**Comment by tcare** [comment]
@curtisman or @digitalinfinity please review

**Comment by curtisman** [comment]
Do we have problem compiling using v150 toolset?

**Comment by tcare** [comment]
__declspec(guard(overflow)) doesn't seem to work when I retarget, and it seems to retarget to v140 anyway. I don't see v150 installed. I didn't spend much time looking.

File: Build/Common.Build.Default.props [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <PlatformToolset Condition="'$(VisualStudioVersion)'=='11.0'">v110</PlatformToolset>
     <PlatformToolset Condition="'$(VisualStudioVersion)'=='12.0'">v120</PlatformToolset>
     <PlatformToolset Condition="'$(VisualStudioVersion)'=='14.0'">v140</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)'=='15.0'">v140</PlatformToolset>
```
Comment (dilijev) [inline Build/Common.Build.Default.props]: v150?

Comment (dilijev) [inline Build/Common.Build.Default.props]: Maybe add a comment explaining why it doesn't match?

**Comment by tcare** [comment]
Ping. New projects in 15 Preview 4 use v140.

**Comment by dilijev** [comment]
@tcare I still favor adding a comment to point out that the `v140` isn't a typo. Otherwise LGTM

**Pull request merged**